app/src/components/trace/TraceTree.tsx

```diff
@@ -158,89 +226,95 @@ function SpanTreeEdgeConnector({
         border-left: 1px solid
           ${statusCode === "ERROR"
             ? theme.colors.statusDanger
-            : "rgb(204, 204, 204)"};
+            : "var(--ac-global-color-grey-700)"};
         top: 0;
-        left: -22px;
+        left: ${nestingLevel * NESTING_INDENT + 29}px;
```

Comment:
some of thee numbers are pretty magical, this is just to get everything to line up correct?

Reply:
so the nesting level is how deep you are in the tree. That automatically pushes the start of the icon in by the amount we want each icon to indent. the 29 makes up for the initial margin (16) + border (4) + half the distance of the icon (10) - 1 for some spacing. I can try to write it out but honestly it's sorta easier to adjust and make sure the math is vaguely right. LMK - happy to make the calcs be variable based.